cfgov/unprocessed/apps/youth-employment-success/js/money.js

```diff
@@ -8,23 +8,24 @@ const _dollarsToPrecisionRegexp = new RegExp( `(\\d+\\.?\\d{0,${ _decimals }})`
  * @returns {Number} Scaled amount in dollars and cents,
  *  fixed to no more than 2 decimal places.
  */
-function _toDollars( dollars ) {
-  const matches = dollars.match( _dollarsToPrecisionRegexp );
+function toDollars( dollars ) {
```

Comment:
Will `toDollars` be public (e.g. exported from the module)?

Reply:
It will be, yes.